pulsar-functions/localrun/src/main/java/org/apache/pulsar/functions/LocalRunner.java

```diff
@@ -166,6 +171,8 @@ public RuntimeEnv convert(String value) {
     protected String secretsProviderClassName;
     @Parameter(names = "--secretsProviderConfig", description = "Whats the config for the secrets provider", hidden = true)
     protected String secretsProviderConfig;
+    @Parameter(names = "--metricsPortStart", description = "The starting port range for metrics server", hidden = true)
```

Comment:
Can you help me understand why it needs to be a range and not just a single port? If it's obvious and I'm simply missing something, perhaps it'd be valuable to describe why it needs to be a range here so that users understand the behavior when using this new feature.

Reply:
There needs to be a range of ports because many function instances can be run via single LocalRun process